run_tests.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+rm -rf cov_profile
+deno task test --coverage=cov_profile 
+deno coverage cov_profile --lcov --exclude=tests/ > cov_profile/coverageFile.lcov
+genhtml -o cov_profile cov_profile/coverageFile.lcov
+open cov_profile/index.html
```

Comment:
This should probably be a `task` in `deno.jsonc` for simplicity's sake. Also, the shell redirect can be replaced with `--output=cov_profile/coverageFile.lcov` if I'm reading the `--help` correctly.